ledger/query/src/query.rs

```diff
@@ -25,16 +26,25 @@ use snarkvm_synthesizer_program::Program;
 
 use anyhow::{Context, Result};
 // ureq re-exports the `http` crate.
-use ureq::http::{self, uri};
+use ureq::http;
+
+mod static_;
+use static_::StaticQuery;
```

Comment:
A publicly-visible struct cannot be imported by its prior path. A `pub use` may have been removed, or the struct itself may have been renamed or removed entirely.

Reply:
Fixed! I made `RestQuery` public too for consistency.